### ISSUE TITLE
git fails to clone repository in WSL

### ISSUE THREAD

**Issue by gomesp** [Microsoft/WSL#3869]
* Your Windows build number:  (Type `ver` at a Windows Command Prompt)

Microsoft Windows [Version 10.0.16299.967]

* What you're doing and what's happening:  (Copy&paste the full set of _specific_ command-line steps necessary to reproduce the behavior, and their output. Include screen shots if that helps demonstrate the problem.)

1. Freshly installed Ubuntu WSL on my box. 
2. Configured git user name, email and generated a new ssh key (and registerd it to GitHub)
3. Tried to clone any git repository

* What's wrong / what should be happening instead:

Git fails to clone a repository, while git on my Windows 10 host (via Git Bash) works perfectly well.

* Strace of the failing command, if applicable:  (If `some_command` is failing, then run `strace -o some_command.strace -f some_command some_args`, and link the contents of `some_command.strace` in a [gist](https://gist.github.com/) here).

```
$ ping github.com
PING github.com (192.30.255.112) 56(84) bytes of data.
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=1 ttl=46 time=174 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=2 ttl=46 time=167 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=3 ttl=46 time=168 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=4 ttl=46 time=165 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=5 ttl=46 time=164 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=6 ttl=46 time=166 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=7 ttl=46 time=164 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=8 ttl=46 time=163 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=9 ttl=46 time=173 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=10 ttl=46 time=165 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=11 ttl=46 time=168 ms
64 bytes from lb-192-30-255-112-sea.github.com (192.30.255.112): icmp_seq=12 ttl=46 time=167 ms
^C
--- github.com ping statistics ---
12 packets transmitted, 12 received, 0% packet loss, time 11012ms
rtt min/avg/max/mdev = 163.933/167.389/174.015/3.289 ms

$ git clone --progress --verbose git@github.com:react-bootstrap/react-bootstrap.git
Cloning into 'react-bootstrap'...
ssh: connect to host github.com port 22: Resource temporarily unavailable
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by sirredbeard** [comment]
Are you using third party security software or anti-virus like Kaspersky?

Are you behind a proxy or VPN?

**Comment by gomesp** [comment]
@sirredbeard I am on a corporate network. My box has only Windows Defender on it, and I have installed the Microsoft Forefront TMG Client (version 7.0.7734.182) on it. I had tried git both with and without a proxy configuration. I don't need a proxy config on the host Windows 10, so I'd assume it should work without the proxy on WSL as well, but I tried it regardless.

**Comment by therealkenc** [comment]
This one was another "resource temporarily available" trying to `ssh` ref #2478 #2723 and friends. Assume the problem is Forefront TMG Client analogy Kaspersky.
